extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -61,7 +61,7 @@ export default function initWADOImageLoader(
       // For now we use image/jls and image/x-jls because some servers still use the old type
       // http://dicom.nema.org/medical/dicom/current/output/html/part18.html
       const xhrRequestHeaders = {
-        Accept: 'multipart/related; type=application/octet-stream',
+        Accept: 'multipart/related; type="application/octet-stream"',
```

Comment:
In addition to Bill's comment, this will conflict in merging in the CSWADO. Why do you need it to be quoted?

Reply:
@sedghi / @wayfarer3130,  Appreciate the diligence with reviewing this. I probably should have clarified better in the issue description why this is needed.
This is currently a Bug in OHIF v3 caused by improperly formatted "accept" header. The encoding of the accept header dictate the "type" to by quoted. 

The fact that this code is needed for OHIF to work with DICOMcloud/MS DICOMweb is because the .NET Framework doesn't accept this improper header.

Ironically, this bug was introduced originally by the DICOM standard in its first DICOMweb revision. Later on, they realized this and updated the standard. We also had to get this fixed in the dcmjs library a while back.

Here is the header RFC of how to encode this: https://www.rfc-editor.org/rfc/rfc2387
Here is the fixed DICOM standard: https://dicom.nema.org/medical/dicom/current/output/html/part18.html#sect_8.7.3.5.1
and the dcmjs issue: https://github.com/dcmjs-org/dcmjs/issues/113

Ultimately, I would like to see the actual streaming of the images decoupled from the wado-loader and off-loaded to something like dcmjs for example so we don't have to fix these kind of issues in multiple places

If you have any questions let me. I think I need to put this explanation on the issue as well :D

Reply:
I understand it technically disagrees with the standard, but doing it the technically correct way causes a significant performance issue when using CORS, because it requires a preflight for EVERY image request.  There isn't a way to do a single preflight for all requests, and it can't be avoided on the sending side except by excluding header values that cause the preflight.  Thus, to make it work reasonably well in other situations, the quotes got removed.

Reply:
Also, the parsing of type allows not quoting the / - it is a bit of a special case, as it changes the parse order a little bit, but doesn't actually require quoting, unlike the @ < and > characters - the specified syntax does NOT have the quotes in it in the RFC, although there is a non-normative note that suggests quoting is required.  My reading of the standard says it technically isn't needed, despite the note.

Reply:
> There isn't a way to do a single preflight for all requests, and it can't be avoided on the sending side except by excluding header values that cause the preflight. Thus, to make it work reasonably well in other situations, the quotes got removed.

@wayfarer3130 are you saying you introduced this bug to to gain performance in CORS? 

`My reading of the standard says it technically isn't needed, despite the note.`
I don't think neither the RFC or DICOM standard are open for interpretation on this issue

Reply:
The rfc standard you referenced actually disallows the quotes - the exact required syntax is: